Bluepill-runner/Bluepill-runner/BPTraceEvent.m

```diff
@@ -0,0 +1,60 @@
+//
+//  BPTraceEvent.m
+//  bluepill
+//
+//  Created by Eric Snyder on 12/17/18.
+//  Copyright © 2018 LinkedIn. All rights reserved.
+//
+
+#import "BPTraceEvent.h"
+
+@implementation TraceEvent
+- (instancetype)init {
+    self = [self initWithData:nil];
+    return self;
+}
+
+- (instancetype)initWithData:(NSDictionary *)data {
+    self = [super init];
+    if (self) {
+        _displayTimeUnit = @"ns";
```

Comment:
us ms?

Reply:
The [spec](https://docs.google.com/document/d/1CvAClvFfyA5R-PhYUmn5OOQtYMH4h6I0nSsKchNAySU/preview) supports both "ms" and "ns". What's the resolution of the timing APIs we're using? We should make sure we are accurate here.

Reply:
I believe the output I'm parsing is in the format of seconds.milliseconds, but I'm not actually sure if the format takes a float, or if I need to convert everything to an integer value of milliseconds, seconds, or what. I'm going to take another quick look at this.